contracts/minter-suite/Minters/MinterMerkle/MinterMerkleV1.sol

```diff
@@ -149,21 +161,36 @@ contract MinterMerkleV1 is ReentrancyGuard, IFilteredMinterMerkleV0 {
     }
 
     /**
-     * @notice Toggles mint limit of one per address for project `_projectId`.
-     * If mint limit is disabled, unlimited mints per address are allowed.
+     * @notice Sets maximum allowed invocations per allowlisted address for
+     * project `_project` to `limit`. If `limit` is set to 0, infinite mints
```

Comment:
Nit: probably don't want to use the word "infinite" without more clarification (only up to allowed # invocations for the project can be minted).

Reply:
good call, updated!